### ISSUE TITLE
Feature: -X --language=

### ISSUE THREAD

**Issue by ricksanchez** [hoovercj/vscode-haskell-linter#20]
It would be nice to be able to add this parameter `--language=`.
Without this, it is annoying to have quasiquoters in a file.
A known work around is to add `{-# LANGUAGE QuasiQuotes #-}` at the beginning of every file where it is needed, but it becomes redundant when using cabal/stack

**Comment by chshersh** [comment]
The same applies to `-XTypeApplications`. `hlint` doesn't work without these settings as well.
It would be really nice to have this feature implemented!

**Comment by elldritch** [comment]
Yes, I just hit this limitation with Arrows. I'd prefer that the extension automatically detect my active extensions from my `.cabal` file, but I'd also be satisfied with the ability to pass the `-X` flag (ideally on a per-project basis, but I could live with a global setting).

**Comment by BiscuitCookies** [comment]
using a .hlint.yaml file in the root of the project allows for this usage

https://github.com/ndmitchell/hlint/blob/master/README.md#customizing-the-hints
